website/templates/emails/welcome.html.mako

```diff
@@ -21,34 +21,34 @@ Files can be stored in a location you specify from the available geographic regi
 % endif
 
 <h4>Store your files</h4>
-Archive your materials, data, manuscripts, or anything else associated with your work during the research process or after it is complete. <a href="https://openscience.zendesk.com/hc/en-us/articles/360019930613?utm_source=notification&utm_medium=email&utm_campaign=welcome">Learn how.</a><br>
+Archive your materials, data, manuscripts, or anything else associated with your work during the research process or after it is complete. <a href="https://help.osf.io/article/353-welcome-to-projects">Learn how.</a><br>
 <br>
 
 <h4>Share your work</h4>
-Keep your research materials and data private, make it accessible to specific others with view-only links, or make it publicly accessible. You have full control of what parts of your research are public and what remains private. <a href="https://openscience.zendesk.com/hc/en-us/articles/360018981414?utm_source=notification&utm_medium=email&utm_campaign=welcome">Explore privacy settings.</a><br>
+Keep your research materials and data private, make it accessible to specific others with view-only links, or make it publicly accessible. You have full control of what parts of your research are public and what remains private. <a href="https://help.osf.io/article/201-create-a-view-only-link-for-a-project">Explore privacy settings.</a><br>
```

Comment:
I don't know what was at this ZD link prior, but would https://help.osf.io/article/285-control-your-privacy-settings be a more appropriate link here? It's more general about privacy settings, rather than specifically about VOLs